HacktoberfestProject.Web/Models/Pr.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace HacktoberfestProject.Web.Models
```

Comment:
Is this so we don't pass the entity model to the view? I thought we decided that it was all public data anyway so we didn't need to create a DTO.

Reply:
The entity model has a load of extra fields from the TableEntity class it inherits from. I was trying to create a barrier that stopped this from happening so the models going to the controller/view only had relevant data.